Modules/Loadable/Markups/qSlicerMarkupsModuleWidget.h

```diff
@@ -365,18 +365,6 @@ public slots:
     qWarning("qSlicerMarkupsModuleWidget::onMissingMarkupPushButtonClicked method is deprecated, please use onMissingControlPointPushButtonClicked instead");
     this->onMissingControlPointPushButtonClicked();
     };
-  /// \deprecated Use onDeleteControlPointPushButtonClicked instead.
-  void onDeleteMarkupPushButtonClicked(bool confirm=true)
```

Comment:
Is the removal of these slots (`onDeleteMarkupPushButtonClicked` and `onDeleteAllMarkupsInListPushButtonClicked`) intentional ? 

It looks like these were initially added in Slicer@516b64a9b and then deprecated in Slicer@edd0af8a5 along with others like `onMissingMarkupPushButtonClicked` ....

Reply:
It’s being replaced by `onDeleteControlPointPushButtonClicked` and `onDeleteAllControlPointsInListPushButtonClicked`. It is due to incorrect names of “Markup” instead of “Control Point” used for button names defined in a UI file.

This means I have missed removing the incorrect name usage of `missingMarkupPushButton` in the UI file as well.

Reply:
I have pushed an update to maintain these deprecated slots, but also still update the object names in the UI file.